website/docs/cdktf/python/r/acm_certificate.html.markdown

```diff
@@ -44,7 +44,7 @@ New certificate materials can be supplied to an existing imported certificate to
 
 ## Private Certificates
 
-Private certificates are issued by an ACM Private Cerificate Authority, which can be created using the resource type [`aws_acmpca_certificate_authority`](acmpca_certificate_authority.html).
+Private certificates are issued by an ACM Private Certificate Authority, which can be created using the resource type [`aws_acmpca_certificate_authority`](acmpca_certificate_authority.html).
```

Comment:
The language specific files are generated automatically from the main file in `/website/docs/r` so this change should be removed.

```suggestion
Private certificates are issued by an ACM Private Cerificate Authority, which can be created using the resource type [`aws_acmpca_certificate_authority`](acmpca_certificate_authority.html).
```